src/main/java/com/github/onsdigital/babbage/api/error/ErrorHandler.java

```diff
@@ -57,6 +58,7 @@ public static void renderErrorPage(int statusCode, HttpServletResponse response)
             response.setStatus(statusCode);
             //Prevent error pages being cached by cdn s
             response.addHeader("cache-control", "public, max-age=0");
+            MetricsFactory.getMetrics().setCacheExpiryTime(0.0);
```

Comment:
needed ?

Reply:
It's only needed if we want to keep track of what the cache expiry time is being set to. If an error occurs then line 60 sets the cache expiry time to 0. We don't have to record this but it might be good to?

Reply:
removed now